org.eclipse.scanning.event.ui/src/org/eclipse/scanning/event/ui/view/StatusQueueView.java

```diff
@@ -135,7 +135,7 @@
 
 	// Data
 	private Map<String, StatusBean>           queue;
-	private boolean                           showEntireQueue = false;
+	private boolean                           hideOtherUsersResults = false;
 
 	private ISubscriber<IBeanListener<StatusBean>>           topicMonitor;
 	private ISubscriber<IBeanListener<PauseBean>>            pauseMonitor;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the "pauseMonitor" field and declare it as a local variable in the relevant methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1450)